### PR TITLE
Set Default ViewMode to Extended

### PR DESCRIPTION
**Pull request by Shailesh351**
Closes https://github.com/WideChat/Rocket.Chat/issues/219

- [x] Set Default ViewMode to Extended
- [x] Added Migration

### CODE REVIEW

File: app/lib/server/startup/settings.js [hunk 1]
```diff
@@ -354,7 +354,7 @@ settings.addGroup('Accounts', function() {
 			public: true,
 			i18nLabel: 'Group_by_Type',
 		});
-		this.add('Accounts_Default_User_Preferences_sidebarViewMode', 'medium', {
+		this.add('Accounts_Default_User_Preferences_sidebarViewMode', 'extended', {
```
Comment (kb0304) [inline app/lib/server/startup/settings.js:357]: This is not necessary, we can leave it as it was.

File: server/startup/migrations/v177.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+} from '../../../app/migrations';
+import {
+	Settings,
+} from '../../../app/models';
+
+
+Migrations.add({
+	version: 177,
+	up() {
+		Settings.updateValueById('Accounts_Default_User_Preferences_sidebarViewMode', 'extended');
```
Comment (kb0304) [inline server/startup/migrations/v177.js:12]: This is not how it is to be done, as we discussed, we'll not change the preference, but rather not use preference and just show extended view when on mobile devices. The preferences can work independently.

**Review by kb0304 — CHANGES_REQUESTED** [review]

File: server/startup/migrations/v177.js (hunk 2 above)
Comment (Shailesh351) [reply server/startup/migrations/v177.js:12]: Some users prefer medium or condensed. So it should be upto their preference?

File: app/lib/server/startup/settings.js (hunk 1 above)
Comment (Shailesh351) [reply app/lib/server/startup/settings.js:357]: If we are setting extended as default, then in the admin options it is used to reset to default mode.

File: server/startup/migrations/v177.js (hunk 2 above)
Comment (Shailesh351) [reply server/startup/migrations/v177.js:12]: https://github.com/WideChat/Rocket.Chat/issues/156 This issue was earlier reported by Jay. Which is regarding adding view mode option in Mobile. So we need to finalize it.

Comment (ear-dev) [reply server/startup/migrations/v177.js:12]: @Shailesh351 I think we have decided to do it differently for now.  Sorry about the confusion.  Karan is suggesting that it defaults to extended on mobile only.  And there is no option to change it in mobile, no preference options for this feature.  But in the web client we would like to leave it as it is upstream.  Thanks.

**Comment by Shailesh351** [comment]
Created separate PR https://github.com/WideChat/Rocket.Chat/pull/229. Closing this one.

**Pull request closed without merging**